app/auth/renku_auth.py

```diff
@@ -21,42 +21,40 @@
 import re
 
 import jwt
-from flask import Blueprint, Response, current_app, redirect, request, session, url_for
+from flask import Blueprint, Response, current_app, redirect, request, session
 
-from .gitlab_auth import GitlabUserToken
-from .web import JWT_ALGORITHM, get_key_for_user
+from .utils import decode_keycloak_jwt, get_redis_key_from_token
+from .gitlab_auth import GitlabUserToken, GL_SUFFIX
+from .web import KC_SUFFIX
 
 # TODO: We're using a class here only to have a uniform interface
 # with GitlabUserToken and JupyterhubUserToken. This should be refactored.
 class RenkuCoreAuthHeaders:
     def process(self, request, headers):
```

Comment:
Could we add some docs here?

Reply:
https://github.com/SwissDataScienceCenter/renku-gateway/issues/232

Reply:
See #232.